src/index.html

```diff
@@ -6,9 +6,98 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img
+         src="./images/logo.png"
+         alt="company logo"
+        >
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+             class="
+                nav__link
+                is-active
+                is-active1"
+                href="#"
+            >
+             apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             class="nav__link"
+             href="#"
+            >
+             samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             class="nav__link"
+             href="#"
+            >
+             smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link"
+               data-qa="hover"
+               href="#"
+            >
+             laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             class="nav__link"
+             href="#"
+            >
+             gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link"
+             href="#"
+            >
+             tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+             class="nav__link"
+             href="#"
+            >
+             photo
+            </a>
+          </li>
+
+          <li class="nav__item item-last">
```

Comment:
You don't use this class in styles, remove it.
```suggestion
          <li class="nav__item">
```